install/kubernetes/cilium/charts/hubble-ui/templates/svc.yaml

```diff
@@ -0,0 +1,53 @@
+{{- if .Values.ui.enabled }}
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: hubble-grpc
+  namespace: {{ .Release.Namespace }}
+  labels:
+    k8s-app: hubble
+spec:
+  type: ClusterIP
+  clusterIP: None
+  selector:
+    k8s-app: hubble
```

Comment:
Since there is no deamonset anymore, this needs to target cilium-agent somehow.

Reply:
Moved it to the cilium/agent/svc.yaml.

Reply:
Addressed review comments.

Reply:
This was actually not addressed, see https://github.com/cilium/cilium/pull/10648#discussion_r408867371

Reply:
Fixed